### DISCUSSION TITLE
some ideas

### DISCUSSION THREAD

**Discussion by ikod** [kerisy/archttp#2] [Ideas]
performance related

- [ ] use async dns resolver for resolving
- [ ] use timerwheel for timers

usability related

- [ ] async http client
- [ ] async redis client
- [ ] async db client
- [ ] `task` api
- [ ] async request termination

**Comment by zoujiaqing** [comment]
Receive!

These suggestions are important.

The current priority is to supplement HttpRequest and HttpResponse apis.
Further performance testing and improvement are then carried out.
Finally, the middleware, DB, Redis, HttpClient, task and other functional modules are improved.

The performance test also needs to be perfected and improved together.

Thanks again ikod ;)
